packages/protocol/contracts/bridge/libs/LibBridgeData.sol

```diff
@@ -27,13 +27,13 @@ library LibBridgeData {
         uint256[45] __gap;
     }
 
-    bytes32 internal constant SIGNAL_PLACEHOLDER = bytes32(uint256(1));
+    bytes32 internal constant MESSAGE_HASH_PLACEHOLDER = bytes32(uint256(1));
     uint256 internal constant CHAINID_PLACEHOLDER = type(uint256).max;
     address internal constant SRC_CHAIN_SENDER_PLACEHOLDER =
-        0x0000000000000000000000000000000000000001;
+        address(uint160(uint256(1)));
```

Comment:
Is there a reason we have to do wrap it 3 times like this?

Reply:
It's the only way to convert 1 to address 0x...1